examples/log-ingestion-k8s/README.md

```diff
@@ -0,0 +1,102 @@
+# Kubernetes Logging with Fluent Bit and Data-Prepper
+
+[Fluent Bit](http://fluentbit.io/) is a lightweight and extensible __Log and Metrics Processor__ that comes with full support for Kubernetes:
+
+* Read Kubernetes/Docker log files from the file system or through systemd Journal
+* Enrich logs with Kubernetes metadata
+* Deliver logs to third party services.
+
+This example uses [minikube](https://minikube.sigs.k8s.io/docs/) to demo Fluent Bit to Data-Prepper pipeline that collects and ingest logs from `my-app` [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/). 
+The service architecture is illustrated as follows:
+![Architecture](K8-fluentbit-data-prepper.png)
+We deploy two `nginx` [pods](https://kubernetes.io/docs/concepts/workloads/pods/) and expose a `my-nginx` [service](https://kubernetes.io/docs/concepts/services-networking/service/) on top of them in the `my-app` namespace. Then in the `logging` namespace,
+we deploy Fluent Bit as [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) that runs in every node in Kubernetes collects and forwards logs to an pre-existing Data-Prepper endpoint, which is located outside 
+Kubernetes cluster.
+
+## Getting started
+
+### Prerequisite
+A Data-Prepper endpoint with [example_log_pipeline.yaml](data-prepper-pipeline-config/example_log_pipeline.yaml) configured.
+
+### Steps
+1. Install [Docker](https://docs.docker.com/get-docker/).
+2. Setup kubectl and minikube locally
+    1. https://kubernetes.io/docs/tasks/tools/install-kubectl/
+    2. https://minikube.sigs.k8s.io/docs/start/
+Notice that this demo uses minikube > 1.21.
+3. Replace the `Host` value in `output-data-prepper.conf` in [fluent-bit-05-configmap.yaml](fluent-bit-05-configmap.yaml) with your Data-Prepper endpoint. For example,
+if the data-prepper is running locally, set `Host` value to be `host.docker.internal`.
```

Comment:
Using `host.docker.internal` is the default for Docker Desktop for Mac, right? Perhaps you could list a few common alternatives, such as for Linux users.